src/components/EditOrganization/index.jsx

```diff
@@ -0,0 +1,208 @@
+import React, { useEffect, useRef, useState } from "react";
+import "./style.css";
+import {
+  Form,
+  Input,
+  Upload,
+  Button,
+  Typography,
+  Divider,
+  Spin,
+  message,
+} from "antd";
+import { InboxOutlined } from "@ant-design/icons";
+import { useTranslation } from "react-i18next";
+import firebase from "../../firebaseConfig";
+import ExampleCoupon from "../ExampleCoupon";
+import "firebase/storage";
+
+const db = firebase.firestore();
+const storage = firebase.storage();
+const storageRef = storage.ref(); // change the storage rule to "request.auth != null" once auth is setup
+const { Title } = Typography;
+const { Dragger } = Upload;
+
+const EditOrganization = ({ isTesting }) => {
+  const { t } = useTranslation();
+  const [organization, setOrganization] = useState({});
+  const [loading, setLoading] = useState(true);
+  const folderName = useRef(null);
+  const fileURL = useRef(null);
+  const fileName = useRef(null);
+
+  useEffect(() => {
+    const unsubscribe = db
+      .collection("restaurants")
+      .where("name", "==", "Arby's") // this needs to come from registration/auth
+      .onSnapshot((snapshot) => {
+        const dataArr = [];
+        snapshot.forEach((doc) => {
+          dataArr.push({ ...doc.data(), docId: doc.id });
+        });
+        setOrganization(dataArr[0]);
+        folderName.current = dataArr[0].name;
+        setLoading(false);
+      });
+    return unsubscribe;
+  }, []);
+  const updateOrgDetail = (data) => {
+    db.collection("restaurants")
+      .doc(organization.docId)
+      .update({
+        name: data.name,
+        website: data.website,
+        description: data.description,
+        email: data.email,
+        logo: `${storageRef}${folderName.current}/${data.logo[0].name}`,
+        logoURL: data.logo[0].url || fileURL.current,
+        logoName: data.logo[0].name,
+      })
+      .then(() => message.success("Successfully updated organization details!"))
+      .catch(() => message.warning("Failed to update organization details!"));
+  };
```

Comment:
please fix indentation in this file there should be a new line between every function and the other